### PR TITLE
EOY 2019 emails

### PR DESCRIPTION
**Pull request by jmbriano**
Hi team, can you please have a quick look at this? It would be nice sending them during this week. 
@loriking @amaliacardenas @raquelalvear @daverick @saljuama

### CODE REVIEW

File: chapter_assets/end-year-mails/2019/end-year-mails.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+Please extend our gratitude to everyone that helped you making this happen at your space!
+
+codebar Barcelona
+barcelona@codebar.io
+
+---
+|Host | Number of events | SPECIFIC PARAGRAPH | IMAGE | SENT |
+|:-:|:-:|:-:|:-:|:-:|
+|Thoughtworks Barcelona|7|During 2019 we doubled the number of workshops at Thoughtworks. You keep being a very special location for codebar in Barcelona. Sharing our mission with you made each of the 7 sessions we had this year a pleasure for everyone. Seeing so many ThoughtWorkers participating in a diversity initiative is encouraging and we could not be more thankful for that. Additionally we'd like to thank you for allowing us to record one of the workshops with the Elastic team|[photo](tws.jpg)|NO|
```
Comment (daverick) [inline chapter_assets/end-year-mails/2019/end-year-mails.md]:
```suggestion
|Thoughtworks Barcelona|7|During 2019 we doubled the number of workshops at Thoughtworks. You keep being a very special location for codebar in Barcelona. Sharing our mission with you made each of the 7 sessions we had this year a pleasure for everyone. Seeing so many ThoughtWorkers participating in a diversity initiative is encouraging and we could not be more thankful for that. Additionally we'd like to thank you for allowing us to record one of the workshops with the Elastic team, it has allow us to get some professional footages of a workshop and we will use it soon to promote codebar.|[photo](tws.jpg)|NO|
```

**Comment by daverick** [comment]
/lgtm
I have added something for TW about the videos

**Comment by jmbriano** [comment]
Status: mails written, reviewed (thanks @loriking and @daverick), and sent. PR merged. 

This card can be moved to closed

**Comment by amaliacardenas** [comment]
@daverick suggested that before we close the issue, we need to update the host rotation list to make sure the contacts are up to date.

**Comment by jmbriano** [comment]
**Status 202003 Monthly meeting**
Document how to keep the contacts list updated. Maybe FMB?

**Pull request merged**